docs/development.rst

```diff
@@ -399,6 +399,26 @@ like e.g. a running MSColab server or a QApplication instance for GUI tests,
 are collected in :mod:`tests.fixtures` in the form of pytest fixtures that can be requested as needed in tests.
 
 
+Changing the database model
+---------------------------
+
+Changing the database model requires adding a corresponding migration script to MSS,
+so that existing databases can be migrated automatically.
+
+To generate such a migration script you can run::
+
+  flask --app mslib.mscolab.app db migrate -d mslib/mscolab/migrations -m "To version <next-major-version>"
+
+Depending on the complexity of the changes that were made,
+the generated migration script might need some tweaking.
+
+If there is already a migration script for the next release,
+then please incorporate the generated migration script into this existing one,
+instead of adding a new one.
+You can still generate a script with the above command first
+to get a starting point for the changes.
+
```

Comment:
ok, that's also an option, then maybe we should have a test that the naming convention is used.  
I assume a model change needs now a mig script or our test setup will cry.

Reply:
> I assume a model change needs now a mig script or our test setup will cry.

Yes, that already happens due to the added tests.

> maybe we should have a test that the naming convention is used.

We would first have to define an actual naming convention. Naming the migration might not even be possible until a release is cut, since the next version is not known at the time of creation of the migration script.